docs/development-setup.md

```diff
@@ -38,7 +38,7 @@ It is also possible to develop directly in an AEA project:
 
 - Create a new project `aea create AGENT_NAME && cd AGENT_NAME`
 
-- Scaffold a new package `aea scaffold --with-symlinks PACKAGE_TYPE PACKAGE_NAME`. This will create the package scaffold under the directory `{PACKAGE_TYPE}s` and create symlinks to ensure package import paths line up with the folder structure. The symlinks are not needed to run the AEA. They are purely for your IDE.
```

Comment:
I just updated this because it was being getting reported as non valid due to PACKAGE_TYPE. I could skip it but since it is *almost* a valid command, I think it's better to make it completely valid and let the validator validate it.